services/movie.py

```diff
@@ -0,0 +1,33 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet[Movie]:
+    queryset = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids,
+                                   actors__id__in=actors_ids)
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)

```

Comment:
The `get_movie_by_id` function uses `Movie.objects.get(id=movie_id)`, which will raise a `DoesNotExist` exception if no movie with the given ID is found. Consider using `get_object_or_404` if this function is used in a view, or handle the exception appropriately to avoid potential crashes.